.prettierrc

```diff
@@ -1,9 +1,9 @@
 {
-  "semi": false,
-  "trailingComma": "es5",
-  "singleQuote": false,
-  "printWidth": 120,
-  "endOfLine": "auto",
-  "arrowParens": "always",
-  "plugins": ["prettier-plugin-tailwindcss"]
+    "printWidth": 120,
+    "tabWidth": 4,
+    "quoteProps": "consistent",
+    "trailingComma": "es5",
```

Comment:
I think this one is nice to have "all" for, makes diffs smaller which is great for review